pandas/core/reshape.py

```diff
@@ -37,9 +37,29 @@ class _Unstacker(object):
 
     Parameters
     ----------
+    values : ndarray
+        Values of DataFrame to "Unstack"
+
+    index : object
+        Pandas ``Index`` or ``MultiIndex``
+
     level : int or str, default last level
         Level to "unstack". Accepts a name for the level.
 
+    value_columns : object, optional
```

Comment:
value_columns: Index, optional
  A MultiIndex must be specified to unstack a DataFrame